airborne_dashboard/components/release/steps/RemoteConfigStep.tsx

```
@@ -0,0 +1,257 @@
+                  )}
+                </div>
+              </div>
+
+              {schemaFields.map((field) => renderRemoteConfigField(field))}
+
+              <div className="mt-6 p-4 rounded-lg">
+                <h4 className="font-medium mb-2">Preview (JSON)</h4>
+                <pre className="hljs text-xs p-3 rounded border overflow-auto max-h-48">
+                  <code
+                    className="language-json"
+                    dangerouslySetInnerHTML={{
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)